Lib/urllib/parse.py

```diff
@@ -662,7 +662,7 @@ def unquote(string, encoding='utf-8', errors='replace'):
 
 
 def parse_qs(qs, keep_blank_values=False, strict_parsing=False,
-             encoding='utf-8', errors='replace', max_num_fields=None):
+             encoding='utf-8', errors='replace', max_num_fields=None, separator='&'):
```

Comment:
The separator parameter should be documented in https://github.com/python/cpython/blob/master/Doc/library/urllib.parse.rst

We should highlight the change in the following manner and add a note about this change.


```
   .. versionchanged:: 3.10
      Introduced *separator* with default value of &

```      
Python versions earlier than Python 3.10 allowed using both ";" and "&" as
query parameter separator. This has been changed to allow only a single
separator key, with "&" as the default separator.